cmd/aws-cloud-controller-manager/controllers/aws_controllermanager.go

```diff
@@ -0,0 +1,50 @@
+package controllers
```

Comment:
This should be in `pkg` instead of `cmd`. Conventionally, `cmd` is to contain packages with `main.go` so that a binary can be generated.

Same for other files in this PR.

Reply:
Also, the file name can be improved - there should be a `_` between words i.e. `aws_controller_manager.go`.

Reply:
got it. I'm moving the files. I used `aws_cloudcontroller.go` because of https://github.com/kubernetes/cloud-provider/blob/master/app/controllermanager.go

I want to have a connection to the intree controller manager to remind us that it is more or less similar functionalities. And I do see that we're already doing similar practice: https://github.com/kubernetes/cloud-provider-aws/blob/master/pkg/providers/v1/aws_instancegroups.go